packages/core/src/tests/TextSplitter.test.ts

```diff
@@ -1,4 +1,6 @@
-import { SentenceSplitter, cjkSentenceTokenizer } from "../TextSplitter";
+import { cjkSentenceTokenizer, SentenceSplitter } from '../TextSplitter'
+// @ts-expect-error
+import { Segment, useDefault as d } from 'segmentit'
```

Comment:
Does this need to be added to the package.json?

Reply:
oh, forgot to remove the testing code